dashboard/pages/fragment_graph.tsx

```diff
@@ -342,38 +334,25 @@ export default function Streaming() {
 
   // Keep the initial snapshot to calculate the rate of back pressure
   const [channelStats, setChannelStats] =
-    useState<Map<string, ChannelStatsDerived>>()
+    useState<Map<string, ChannelDeltaStats>>()
 
   const [fragmentStats, setFragmentStats] = useState<{
     [key: number]: FragmentStats
   }>()
 
   useEffect(() => {
-    // The initial snapshot is used to calculate the rate of back pressure
-    // It's not used to render the page directly, so we don't need to set it in the state
     let initialSnapshot: ChannelStatsSnapshot | undefined
 
-    function refresh() {
```

Comment:
Just reuse the refactored function above: https://github.com/risingwavelabs/risingwave/pull/22838/files#r2266136364